packages/css-library/dist/tokens/scss/variables.scss

```diff
@@ -1,15 +1,13 @@
 
 // Do not edit directly
-// Generated on Tue, 08 Aug 2023 21:11:38 GMT
+// Generated on Fri, 25 Aug 2023 20:50:56 GMT
 
 $color-base: #212121;
 $color-white: #ffffff;
 $color-black: #000000;
-$color-orange: #eb7f29;
 $color-link-default: #004795;
 $color-link-visited: #4c2c92;
-$color-warning-message: #fac922;
-$color-gibill-accent: #fff1d2;
+$color-feedback-warning-background: #fac922;
```

Comment:
When we eventually change-over from Formation to CSS Library, I wonder how we will deal with this token name being changed. It's being used in a few places in Formation as well as vets-website.

- Formation: https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fveteran-facing-services-tools%20path%3A%2F%5Epackages%5C%2Fformation%5C%2F%2F%20%24color-warning-message&type=code
- vets-webste: https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fvets-website%20%24color-warning-message&type=code